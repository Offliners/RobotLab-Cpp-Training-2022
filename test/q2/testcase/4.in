438
sighTSEEing
AWkWARDLY
AroUse
tHUnDEr
SPECIFIC
AEStHETiC
FaNTasy
mOTHerhoOD
BAcTeRIA
BUtCHER
us
imPersonal
mouse
rOMANTIC
FAIR
BALcONY
level
bEe
coVER
INTeRRUpT
prosperous
PrEFeRAbLY
INveStIgAToR
scoLd
IDEOlOGY
darkeN
EACH
cRItIc
IDENTITy
LIgHT
nOODLE
swallow
cOMB
LANDLADY
STUFF
aShamed
DRy
adDReSS
t3rougho2t
tHerMal
numErICAL
eROSion
manufaCTUrer
FRUSTRATE
i.q.
haRsH
HONOR
eNviSAgE
VIEW
consuMer
MARgIN
outline
SPECIAL
BROADCAST
lAnGUAge
EXCLaIM
gRANdsON
luggage
sufFICiEnt
BRiSK
advise
gum
rOCKET
PEACe
03U9r8
CoUSIN
MINE
Panic
HarbOr
jUsT
crowd
MisEry
responD
cHeW
FORwaRd
TAle
INTERMEDIATE
rEsearcher
DeNoMINATIOn
CHOIR
PLANT
CAPTIVE
HOmeLy
necktie
eAgerlY
MUSCULAR
sCHOlaR
S8134A9T67
mInimum
diNE
MIRROR
MONEY
GRAVEyARD
medIUm
pHysicIST
learNEd
ElEmEnt
wAYSIDE
cALOrie
glamour
negOTiatE
DIp
revisioN
TOrture
aMaTEur
INTERACTION
enORMoUS
PEAR
pRoTEsT
s
RECoNSTRUCtiON
CELeBritY
clAY
premIer
unIVerSiTy
dEcoraTe
goLDeN
PERHAPS
bump
CONSOLE
shoCKEd
diabetEs
inTImIDate
FAREWELL
HOm
tresPass
juDicial
CrasH
NEVEr
creDIbLe
COnNeCTiON
PRONoUncE
guIdELINe
UNFAIR
recRuIT
conVerTIble
coVErAgE
gLoomY
CROwN
HEAL
iNCENTIVE
COAST
reluctant
GORgEOuS
chARaCTer
helpFul
sEduce
TOOTHBRUSH
idIom
COLlECtOr
counteratTack
foot
moIst
HOliDAY
cOMMUniCAte
pathetic
EVER
pRActice
BaseMent
eXIStEnCE
aCreAge
IMpORTANCE
POrt
fUnD
hUrt
cemetery
clASSiC
insTiTUTioNaL
eXpErimentAtioN
dowNtown
BORE
reStAurAnt
0Y2L4
aiRline
WeiGht
Hyg36ne
tailoR
greasy
EcOnOmIst
HoweVER
THING
CArToON
wEB
MANdaRIN
IMMIGRATION
BLInk
oRDinaNCe
stARt
creature
iNvOlve
sLaSh
PilL
flow
INitIAL
ARISE
offeNsIve
ErRand
GODdess
extEnSIon
H7LiDA0
piMpLE
pOLlUTIoN
FORMAL
FR2Wn
distiNGuIsh
SKIllfUL
coLor
ACc1M5L2s5
humorously
p
hurrah
SANITATION
dessERt
snEAker
CoUrT
statuRE
taNK
ACCEPT
sUPPRESs
DWELLER
become
EnvElOPE
comPaTIBLy
HEAT
CONScIOuSNeSS
fAN
StEreO
TechnIQUe
strong
STREET
NumeRoUS
ACqUIRE
CIvIcS
cOCKpIt
harassment
LeaDer
JOURNEY
TENSION
relEASe
bIoCheMistrY
FAVORABLE
pRiNcE
until
TOlEratE
piGeoN
CRACK
oBtAin
r5be
stATe
assign
latteR
thigh
8802E5UL
deservE
OdDS
SwAN
USE
leAThER
rIDDLE
garage
pOTtER
18397mA7
ChATTERbOX
caTER
COUcH
touch
DISPaTCH
pAssIvE
DISQuaLIFY
cOnsTraInT
BAStARD
afFord
FORESt
judGE
BLamEwOrTHY
amazE
PAStuRE
nATIon
herALD
hANGEr
SuRgERY
fuR
OBservaTion
harden
Vine
l121y
cLUB
regimeNt
pOnY
geNErouS
ineffective
grow
DELIghT
aNiMAl
1A7
aLgebra
NoMINATIon
OrIEntaL
altHouGh
ASSISt
cuRrEnt
pARLoR
SPoRT
WATERMELON
outbReak
COMPATIBLE
music
MELOdY
MUd
pressuRe
TamE
cONveY
BID
hETErOSEXUAL
trAGiC
tUNnEl
cOmpuTERizE
282p
goAl
iGNoRE
465I8U5
aVAilability
REcolLecT
tRousers
FREEZe
distRIBuTE
gap
patTern
dignitary
miNer
greed
MayoR
lonesOme
naive
qUaLIFY
nEedle
pHilOSOphY
CHAnCelloR
VeRSE
aDditioN
up
jUmp
APPEAL
maChinE
SORT
OwN
CALeNdAr
PROFICIENT
bOx
DUKE
eaRNINgS
gigantic
EXPlICIT
NArRaTE
monetary
cOmmunAL
taX
sociETY
EAT
RecoGNITiON
PoTTeRY
suPEriOr
ARmY
SIBliNG
BUDGET
LAUGHteR
tRAY
cowArDICe
pAste
CLOtH
4CCU65
I
bLur
GratiTUDe
O9E1coat
hUmILIAtiON
ExAmPlE
iNsTabilItY
saLvatIoN
lengtHeN
TROPICAL
sWoRD
vENUS
VIOLin
locatE
n2tr793o4
UnIFy
BREAKFAST
UNDERsTANDaBLe
grEase
swim
MErCHaNT
sheep
THOUgH
meDICATIOn
aPPETITE
aSIDE
acceptance
SNAP
loviNG
steer
I3Te28IoN
ConSPiracY
DoLlaR
habItuAl
ChUrCHYARD
SHEeP
3O19TGU8RD
ESCAPE
KNEE
C5895
02S94OY
villa
6IRC5S
spiNACH
MANage
albuM
InFOrm
gRApeFRUIT
FUrIOUs
politics
fat
RESPONsIBilIty
symmEtRy
hYpOtHEsIs
kindness
VoGUe
telegRaM
AUTumN
GOAlKEEper
ORDINAN6E
L
scanDaL
BUTTeRFLY
DEVIATION
TABLEsPOON
cOmMeND
maTTer
