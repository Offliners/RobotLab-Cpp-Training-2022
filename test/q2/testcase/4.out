n. 骨骸
Unknown
n. 天空
n. 衛星
n.  鴿
n. 潛在性
n. 慶祝
adj. 不朽的
n. 生薑
n. 缺口；峽谷；隔閡
Unknown
adj.  天真的；樸素的
n./v. 拘留所
n./v. 價格
v. 屈辱
v. 裝船；從事
adj. 免除的
n. 熱情
n. 馬拉松賽跑
n./v. 壓力
n./v. 聊天；說話
adj. 現代的
adj./n./pron. 少數
adj. 戰略(性)的
n. 陸標
adj. 防護的；保衛的
adj. 仁慈的
n. 棗；棗椰樹
n. 罐；壺；盒
n. 項鍊
n. 售貨棚
v. 溝通
n. 葡萄園
n. 所有物
n. 設計師
n. 語言學
n. 處理；配置
n. 教師
adj. 複雜的
adj. 小心謹慎的
n. 行政官員
n./v. 點頭；打瞌睡
n. 灰燼；骨灰
Unknown
Unknown
v. 原諒
n. 幾何學
n. 手提箱
n. 方言
n./v. 銹；生銹
n. 家庭作業
v./n. 遭遇；衝突
n. 模型；模範
adj. 種族的
v. 隱約的或威脅性的出現
v. 違犯
n. 生產力
adj./n. 否定的
n./v. 小船；划船
n. 椰子
adj. 統計(學)的
adj./n. 慣常的
adj. 傳統的
n. (被)起訴
n. 事實
n. 拖鞋
n. 電影
n. 護民官
n. 技巧
n. 連接；關聯
n. 對質；對抗
n. 地毯
n. 隊
adj./n. 軍事的；軍隊
n. 敵人
n. 高度
n. 蛋白質
n. 重力；萬有引力
n. 安排；佈置
n. 作曲家；調停者
v. 結束；斷定
n. 教導；指示
v. 消耗；花費
v. 珍惜
n./v. 野餐；去郊遊野餐
n. 斧頭
n. 噪音
n. 到來
adj./adv. 一週一次的
n./v. 群；聚集
adj./adv. 位於兩地中途的
n. 花生
n./v. 手；交給
n. 地震
n. 麵團
v. 違抗
n. 藥膏
v./n. 打中；打擊
v. 否認
n. 保險套
adj. 〔數〕二項式
n. 奔跑的人
adj. 輕率的
adj. 在內的
n. 光碟
n./v. 汗水；出汗
v. 同時共存
adj./adv. 靜止的；仍然
prep. 在…當中
adj./adv./v. 進一步的
n. 爭吵
v. (n)完成；達到
adv. 在旁邊；除…之外
v. 鼓舞
n. 火雞
v. 把…弄糊塗；攪亂
n. 工作
n. 洗禮
n. 宣稱
v. 把…歸咎(於)；指控
n. 櫻桃
n. 獨處(狀況)
n. 居先；優先(權)
n. 家禽(肉)
n. 成年；男子氣質
n./adj. 化妝品；美容的
v. 限制；使侷限
n. 鼠
adj. 簡單的
n. 人口
adj. 個人的
adj./adv. 早的
n. 樹脂；口香糖
n. 敵意
adj./n. 最初的；原著
v. 徘徊(於)
n. 報告者
v. 使富裕
v. 批准
adj. 野心勃勃的
adj. 相似的
n. 麵包
Unknown
n. 反映
v./n. 減少
n. 小馬
adj. 有生氣的
n. 保險費
v./n. 舉起；上漲
n. 舌」鬍刀
Unknown
v. 慶祝
adj. 不合常規的
n. 丈夫
n. 檢查
adj. 主觀的
adv. 兩次
n. 夜
adj. 理論(上)的
n. 顧問；諮詢者
adj. 無窮的
n. 牛；牲口
n./v. 打破；破曉
adj. 違法的
n. 攻擊
v. 有…的傾向　
v. 發強烈的光
adj. 精確的
adj. 陰險的
n./v. 碼頭
pron. 本身
v./n. 作弊；欺騙
v. 屈辱
n. 好萊塢
adj. 有禮貌的
n./v. 答覆
adj. 習慣上的
adj. 類似的
v. 收集；聚集
adj. 核子的
v./n. 爭論
n./adj. 批發；大批的
n. 助理
n. 護送(車隊)
n. 觸鬚；天線
v. 指導
n. 經濟學者
n. 液體
n. 窗玻璃片
n. 親切
conj. 除非
n. 鎳
n. 碎片
